pkgs/top-level/php-packages.nix

```diff
@@ -486,6 +501,8 @@ in
 
       nativeBuildInputs = [ pkgs.pkgconfig ];
       buildInputs = with pkgs; [ cyrus_sasl zlib ];
+
+      meta.maintainers = pkgs.lib.teams.php.members;
```

Comment:
It should be possible to remove `pkgs` from all of these, since `lib` is already in scope.